tests/data/import_files/array-api-extra.yml

```diff
@@ -0,0 +1,5 @@
+name: array-api-extra
```

Comment:
These files are pretty short.
I wonder if it would make the tests more readable if you'd just create within the tests.